patchwork-events-lifecycle/src/main/resources/patchwork-events-lifecycle.mixins.json

```diff
@@ -11,6 +11,9 @@
     "MixinClientWorld",
     "MixinMinecraftClient"
```

Comment:
Sorry, missed the boat on this PR. Shouldn't MixinIntegratedServer be included here? I can't see it in any of the Mixin JSON files.

Reply:
Good catch! I will fix right now.